x-pack/plugins/apm/public/components/fleet_integration/apm_agents/replace_template_strings.ts

```diff
@@ -8,16 +8,11 @@
 import { CoreStart } from 'kibana/public';
 import Mustache from 'mustache';
 
-const TEMPLATE_TAGS = ['{', '}'];
-
```

Comment:
Those templates tags should still be there to support the `config.docs.*` variables with single brackets in the fleet instructions: https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/common/tutorial/instructions/apm_agent_instructions.ts#L43. The default is double brackets.